packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -91,6 +91,13 @@ export const navigationRoutes: NavigationRoute[] = [
     },
     disabled: true,
```

Comment:
Let's test performance impact of syncing all css variables from tailwind.

If it's a problem - let's add a solution for users to fix that.

Reply:
I haven't faced any performance issues, except lots of css variables in the code.